test/utils/server_version.hxx

```diff
@@ -45,38 +45,44 @@ struct server_version {
 
     [[nodiscard]] bool is_cheshire_cat() const
     {
-        // [7.0.0, inf)
-        return major >= 7;
+        // [7.0.0, 7.1.0)
+        return major >= 7 && minor < 1;
```

Comment:
== 7 ?

Reply:
let it me >= until we get next version defined

Reply:
ah, right. sorry